notebooks/en/agent_data_analyst.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>Solve the Titanic sounds a bit off, maybe reword it and add some context!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>